examples/cartesian_reconstruction_3d.py

```diff
@@ -0,0 +1,99 @@
+"""
+Neuroimaging cartesian reconstruction
+=====================================
+
+Author: LElgueddari
+
+In this tutorial we will reconstruct an MRI image from the sparse 3D kspace
+measurements.
+
+Import neuroimaging data
+------------------------
+
+We use the toy datasets available in pysap, more specifically the 3D orange
+and the cartesian acquisition scheme.
+"""
+
+# Package import
+from modopt.math.metrics import ssim
+from mri.numerics.fourier import FFT
+from mri.numerics.reconstruct import sparse_rec_fista
+from mri.numerics.utils import generate_operators
+from mri.numerics.utils import convert_mask_to_locations
+from pysap.data import get_sample_data
+import pysap
+
+# Third party import
+import numpy as np
+
+# Loading input data and convert it into a single channel using Sum-Of-Squares
+image = get_sample_data('3d-pmri')
+image.data = np.sqrt(np.sum(np.abs(image.data)**2, axis=0))
+
+# Obtain K-Space Cartesian Mask
+mask = get_sample_data("2d-poisson-disk-mask")
+mask.data = np.repeat(np.expand_dims(mask.data, axis=-1), 160, axis=-1)
```

Comment:
Can we try and save the complete mask directly? I feel it would be cleaner code than having even a part of mask preparation in the example. I feel its good to follow the same flow as is done in other examples. Also, the example must be, in my opinion all about how to reconstruct.